projects/js-packages/publicize-components/src/components/notice/styles.module.scss

```diff
@@ -0,0 +1,15 @@
+.notice {
+	color: #757575;
+	margin-bottom: 16px;
+}
+
+.notice--warning {
+	padding: 8px 16px;
+	color: #4f3500;
+	background-color: #f5f1e1;
+	border-left: 2px solid #f0c930;
```

Comment:
Can we leverage `@automattic/jetpack-base-styles` for some of the values here?

Reply:
Is that possible? I wasn't sure to what extent we could use our existing components in the Gutenberg editor. Perhaps I can wrap it in a [`ThemeProvider`](https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/components/components/theme-provider) and get the styles like that.

Reply:
👍 You can check how we use this in other blocks for some examples.

Reply:
Fixed in [c424135](https://github.com/Automattic/jetpack/pull/25661/commits/c424135adb3b9052e4e61183b3ad580254ad7f7b).